libjava/lib/src/main/java/com/mapbox/services/commons/MapboxService.java

```diff
@@ -14,6 +14,8 @@
  */
 public abstract class MapboxService<T> {
 
+    public final static String HEADER_USER_AGENT = "User-Agent: MapboxAndroidServices/1.0";
```

Comment:
Is there a way to dynamically update the version to match the library’s actual version? Otherwise, the source code should be updated as part of the build or packaging process.